examples/sandbox.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: agents.x-k8s.io/v1alpha1
 kind: Sandbox
 metadata:
   name: sandbox-example
+  namespace: sandbox-ns
```

Comment:
for this we need to create a sandbox-ns namespace as well in this file. 

The goal of this example is for someone to run kubectl apply -f examples/sandbox.yaml and get a working sandbox (in default ns).

If we want your changes to work we need to add ns and sa in this file as well.

Reply:
Ahhh makes sense, okok, let me add those two!